src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -258,6 +258,12 @@ public void CB_COMMANDER_CREATE(LoginConnection conn, Packet packet)
 			character.Spr = jobData.Spr;
 			character.Dex = jobData.Dex;
 
+			if (!character.SetDefaultEquipment())
```

Comment:
What happens to the client in this case? Is it okay with not receiving a response? And if not, is there a way to recover from the mistake of the server, or should the connection maybe be killed to get them out of there?

Reply:
The result is that the client would not progress away from the character creation screen. I see that this should at the very least reply with `BC_MESSAGE` to inform the user about why the action did not work.